Example/android/app/src/main/java/com/swmansion/rnscreens/example/MainApplication.java

```diff
@@ -5,6 +5,8 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.reactnativerestart.RestartPackage;
```

Comment:
Same here about `Restart`

Reply:
FIxed in https://github.com/software-mansion/react-native-screens/pull/827/commits/4af99c217e668f87d6ab67bfb8f04773487448c4